R/tm_g_butterfly.R

```diff
@@ -252,31 +252,20 @@ ui_g_butterfly <- function(id, ...) {
         value = a$legend_on
       )
     ),
-    forms = get_rcode_ui(ns("rcode")),
+    forms = teal.widgets::verbatim_popup_ui(ns("rcode"), "Show R code"),
     pre_output = a$pre_output,
     post_output = a$post_output
   )
 }
 
-srv_g_butterfly <- function(id, datasets, reporter, dataname, label, plot_height, plot_width) {
+srv_g_butterfly <- function(id, data, filter_panel_api, reporter, dataname, label, plot_height, plot_width) {
```

Comment:
A validation message should show up when Right or left dich. levels are not selected:
![Screenshot from 2022-09-07 15-27-47](https://user-images.githubusercontent.com/86970066/188890526-4b6197d4-7cad-4df8-ab23-3f8427d2df89.png)

Reply:
I have checked the app and I can't confirm this - I checked early-dev and app from butterfly's roxygen

![image](https://user-images.githubusercontent.com/6959016/189055994-0548dace-1409-4513-98d6-3d09d0d34fc2.png)

Reply:
fixed 👍